### ISSUE TITLE
Error when trying to submit Pricing Feed

### ISSUE THREAD

**Issue by ej-cdx** [saleweaver/python-amazon-sp-api#340]
**Describe the bug**
I get an error when I try to submit a new pricing XML using submit_feed(). I have been using version 0.5.1 and this was not an issue, but as soon as I started using 0.12.1, this issue came up. It seems like a recent change triggered this.

**Error Message**
```
File "/Users/xxxx/.venv/lib/python3.10/site-packages/sp_api/api/feeds/feeds.py", line 173, in create_feed_document
    data=file.read().decode('iso-8859-1'),
AttributeError: 'str' object has no attribute 'decode'. Did you mean: 'encode'?
```

***/!\ When I change feeds.py to encode, feed submission works and there is no more error. ***

**To Reproduce**

Here is the code I use to submit this pricing feed.
```
with open(xmlpath, "r") as xml_file:
        res = Feeds(account=account).submit_feed(
            feed_type="POST_PRODUCT_PRICING_DATA", file=xml_file
        )
    print(res[0])
    print("API feed submitted ----")
```

**Expected behavior**

After submitting XML feed, it is expected to get a payload with feedDocumentId. 

**Desktop (please complete the following information):**

Mac OS 
VS Code for Mac

**Comment by saleweaver** [comment]
Hi,

can you try to `with open(xmlpath, "rb") as xml_file:` please.

**Comment by ej-cdx** [comment]
> Hi,
> 
> can you try to `with open(xmlpath, "rb") as xml_file:` please.

It does work now. Thank you!

**Comment by saleweaver** [comment]
I've just released v0.12.2, you can use byte or string mode now
